services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/document/DefaultEnhancedDocument.java

```diff
@@ -403,6 +403,8 @@ public Builder putJson(String attributeName, String json) {
 
         @Override
         public Builder remove(String attributeName) {
+            Validate.isTrue(!(attributeName == null || attributeName.trim().length() == 0),
```

Comment:
Nit: can we use `StringUtils.isEmpty()`?

Reply:
done